src/aks-preview/HISTORY.rst

```diff
@@ -15,6 +15,7 @@ Pending
 
 0.5.139
 * `az aks create` and `az aks nodepool add`: Add warning message when specifying `--os-sku` to `Mariner` or `CBLMariner`.
+* `az aks create` and `az aks enable-addons`: Change the default value of `--enable-msi-auth-for-monitoring` to `true` and add check for airgap clouds for monitoring addon
```

Comment:
Would be better to release a new version of aks-preview. Please move this block several lines above, choose a new version number, and update the version number in setup.py accordingly.

Reply:
thank you for the suggestion, changed to use 0.5.140